WordPress/Classes/ViewRelated/Views/AlertView/AlertInternalView.swift

```diff
@@ -0,0 +1,63 @@
+import Foundation
+
+
+/**
+*  @class           AlertInternalView
+*  @brief           Helper class, used internally by AlertView. Not designed for general usage.
+*/
+
+public class AlertInternalView : UIView
+{
+    // MARK: - Public Properties
+    public var onClick : (() -> ())?
+    
+    
+    
+    // MARK: - View Methods
+    public override func awakeFromNib() {
+         super.awakeFromNib()
+        
+        // Check the Outlets
+        assert(backgroundView   != nil)
+        assert(alertView        != nil)
+        assert(titleLabel       != nil)
+        assert(descriptionLabel != nil)
+
+        // Setup please!
+        alertView.layer.cornerRadius    = cornerRadius
+        titleLabel.font                 = Styles.titleRegularFont
+        descriptionLabel.font           = Styles.detailsRegularFont
+
+        titleLabel.textColor            = Styles.titleColor
+        descriptionLabel.textColor      = Styles.detailsColor
+
+        dismissButton.titleLabel?.font  = Styles.buttonFont
```

Comment:
We should try to avoid aligning all code like this, because when the code is modified it creates the need to maintain the perfect alignment (ie: more time spent than necessary).  I'm taking this example but it can be found in other places in this PR (see asserts above and outlets a few lines below).

Keep in mind the reader of your code should be able to understand each of these lines regardless of alignment.